games-strategy/freeciv/freeciv-2.5.5.recipe

```diff
@@ -99,6 +101,18 @@ INSTALL()
 
 	mkdir -p $appsDir/Freeciv
 	mv $binDir/freeciv-sdl $appsDir/Freeciv/Freeciv
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local INTERNAL="`echo "$portVersion" | cut -d. -f4`"
```

Comment:
Maybe we could hard-code the 0 for internal.
(Most pacakges only have x.y.z versions.)